libc/src/__support/CPP/atomic.h

```diff
@@ -166,7 +166,7 @@ LIBC_INLINE void atomic_thread_fence([[maybe_unused]] MemoryOrder mem_ord) {
 // except no instructions for memory ordering are issued. Only reordering of
 // the instructions by the compiler is suppressed as order instructs.
 LIBC_INLINE void atomic_signal_fence([[maybe_unused]] MemoryOrder mem_ord) {
-#if __has_builtin(__atomic_signal_fence)
+#if LIBC_HAS_BUILTIN(__atomic_signal_fence)
```

Comment:
`constexpr if` would be a good fit here as well.

Reply:
it seems that clang would not be happy with unknown function in `constexpr if`: https://godbolt.org/z/xeGevdqzW

Reply:
Ah, yeah. I remember running into that because `constexpr` is only evaluated after the AST is built.